### PR TITLE
Modernize code to Python 3.6+

### PR DESCRIPTION
**Pull request by eumiro**
Remove some redundant code that is covered by defaults in Python 3.

And I had to include `README.md` to `MANIFEST.in`. Without that, `tox` failed with an error, that the file `'./README.md` could not be found. Wondering why nobody else had that problem.

### CODE REVIEW

File: fpdf/fpdf.py [hunk 1]
```diff
@@ -113,7 +112,7 @@ def load_cache(filename):
         with open(filename, "rb") as fh:
             return pickle.load(fh)
     # File missing, unsupported pickle, etc
-    except (IOError, ValueError):
+    except (OSError, ValueError):
```
Comment (Lucas-C) [inline fpdf/fpdf.py:114]: 👍

**Review by Lucas-C — APPROVED** [review]
Thanks for the contribution!

I do not use ``tox` much myself, but thank you for taking the time to submit a fix!

**Comment by eumiro** [comment]
And, as a bonus, I added the introduction of f-strings (we're at Python 3.6+). Please let me know if you'd like to see them in a separate PR.

**Comment by Lucas-C** [comment]
I see you are still adding commits to it :)

Take your time, and drop a comment when this is ready for review 😉

**Comment by Lucas-C** [comment]
Adding f-strings as part of this PR is fine

**Comment by eumiro** [comment]
I think the f-strings commit was too much for this PR, sorry for the CI fails e-mail flood. I will drop it from this PR and start a new one after you've merged this one.

Now the tests are green, so I am done with this PR.

**Comment by Lucas-C** [comment]
Merged! Thank you very much for your contribution :)

**Pull request merged**